cpp/tests/io/orc_test.cpp

```diff
@@ -114,6 +114,15 @@ struct OrcChunkedWriterNumericTypeTest : public OrcChunkedWriterTest {
 // Declare typed test cases
 TYPED_TEST_CASE(OrcChunkedWriterNumericTypeTest, SupportedTypes);
 
+// Test fixture for reader tests
+template <typename T>
+struct OrcReaderTest : public cudf::test::BaseFixture {
+  auto type() { return cudf::data_type{cudf::type_to_id<T>()}; }
+};
+
+// Declare typed test cases
+TYPED_TEST_CASE(OrcReaderTest, SupportedTypes);
```

Comment:
Does this mean that the new tests are executed per type? I don't think this is necessary given the bug/fix is unrelated to types.